packages/url/src/filter-url-for-display.js

```diff
@@ -19,5 +19,26 @@ export function filterURLForDisplay( url ) {
 		return filteredURL.replace( '/', '' );
 	}
 
+	// Prettify image urls
+	const mediaRegexp = /([\w|:])*\.(?:jpg|jpeg|gif|png|svg)/g;
```

Comment:
All tests pass if I change this to...

```suggestion
	const mediaRegexp = /\.(?:jpg|jpeg|gif|png|svg)/g;
```

I'd wonder then:

1. Can we remove it?
2. Or, are we missing a test case to protect against whatever its purpose for existing is?

Reply:
It looks safe to check only for extensions but that would include all kinds of media extensions (audio and video too)